src/main/resources/templates/index.html

```diff
@@ -6,35 +6,35 @@
     <title>FindMyRecycling</title>
 </head>
 <body>
+    <div class="wrapper">
+        <header class="header">
+            <h2>FindMyRecycling</h2>
+        </header>
```

Comment:
divs won't do for blind people with listening devices!

Reply:
Good.  Structuring content with proper HTML is better for accessibility... and numerous other reasons!